src/router/index.js

```diff
@@ -39,6 +38,11 @@ const routes = [
     name: "ExamEnvironment",
     component: () => import("@/pages/exam-environment/ExamEnvironment.vue"),
   },
+  {
+    path: "/exam-result",
```

Comment:
```suggestion
    path: "/exam-result/:examResultId",
```
나중에 API 연결할 때, examResultId가 있어야 쉽게 불러올 수 있습니다...!
라우팅 같은 부분은 웬만하면 기존 코드를 유지해주시구, 바꿔야할 때는 한번 물어봐주세요!